Browser_IDE/editorMain.js

```diff
@@ -1,39 +1,36 @@
 "use strict";
-
+let executionEnviroment = new ExecutionEnvironment(document.getElementById("ExecutionEnvironment"));
+let storedProject = new IDBStoredProject(makeNewProject);
 // ------ Setup UI ------
 
-function setupCodeArea(element){
-    let editor = CodeMirror.fromTextArea(element, {
-        mode: "text/javascript",
-        theme: "dracula",
-        lineNumbers: true,
-        autoCloseBrackets: true,
-        styleActiveLine: true,
-        extraKeys: {"Ctrl-Space": "autocomplete"},
-        hintOptions: {
-            alignWithWord: false,
-            completeSingle: false,
-            useGlobalScope: false,
-        },
-        foldGutter: true,
-        gutters: ["CodeMirror-linenumbers", "CodeMirror-foldgutter"]
-    });
-
-    editor.on('inputRead', (cm, change) => {
-        if (!cm.state.completeActive) {
-            cm.showHint();
-        }
-    });
-    return editor;
+let editors = []
+let editorPathMap = {};
+function addNewEditor( codeblock_name,eleID,ExecutionEnvironment, StoredProject,filePath  ) {
+    let parts = filePath.split("/");
+    let filename = parts[parts.length - 1];
+    let elementID = document.getElementById(eleID);
+    if (!elementID) {
+        // Create new element if it doesn't exist
+        elementID = document.createElement('div'); // Create a new div element
+        elementID.id = eleID; // Set the id of the new element
+        document.body.appendChild(elementID); // Append the new element to the body of the document
```

Comment:
This will simply add the `div` to the end of the page. It needs to be inserted within the container for code editors.

You should test this by removing the hard-coded editor elements from the `index.html` file.